addons/l10n_it_edi/wizard/account_move_send.py

```diff
@@ -47,55 +42,38 @@ def _get_wizard_vals_restrict_to(self, only_options):
     # COMPUTE/CONSTRAINS METHODS
     # -------------------------------------------------------------------------
 
-    @api.depends('l10n_it_edi_actionable_errors')
-    def _compute_l10n_it_edi_warning_message(self):
-        # To be removed -- Proxy feature to be replaced with actionable_errors as soon as the user updates the module
-        for wizard in self:
-            messages = []
-            wizard.l10n_it_edi_warning_message = False
-            if wizard.l10n_it_edi_actionable_errors:
-                messages.append(_("Please upgrade the Italian EDI module to update this widget."))
-                messages.append(_("Go to Applications page and update the 'Italia - Fatturazione Elettronica' module."))
-                messages.append("")
-                for error_key, error_data in wizard.l10n_it_edi_actionable_errors.items():
-                    message = error_data['message']
-                    split = error_key.split("_")
-                    if len(split) > 1 and (model_id := {
-                        'partner': 'res.partner',
-                        'move': 'account.move',
-                        'company': 'res.company'
-                    }.get(split[0], None)):
-                        if action := error_data.get('action'):
-                            if 'res_id' in action:
-                                record_ids = [action['res_id']]
-                            else:
-                                record_ids = action['domain'][0][2]
-                            records = self.env[model_id].browse(record_ids)
-                            message = f"{message} - {', '.join(records.mapped('display_name'))}"
-                    messages.append(nl2br(escape(message)))
-                wizard.l10n_it_edi_warning_message = Markup("<br/>").join(messages)
+    @api.depends('l10n_it_edi_enable_send')
+    def _compute_send_warning_message(self):
+        # EXTENDS 'account'
+        super()._compute_send_warning_message()
+        for wizard in self.filtered('l10n_it_edi_enable_send'):
```

Comment:
@lordkrandel May I ask for a quick look to Italian changes on this PR please ? Since they're related to things that should have been removed 🙂
On this particular line, if I'm not mistaken, we were previously computing `actionable_errors` when the company had fiscal code to Italy. Yoni suggest to switch this to `l10n_it_edi_enable_send`. I'd even switch it to "if one of the two checkboxes is checked" to avoid spamming with unnecessary warnings.
What do you think ?